### ISSUE TITLE
slf4j+jul not worked on payara 4.1.1.161

### ISSUE THREAD

**Issue by niblings** [payara/Payara#651]
The difference of the output to the "server.log"

Payara 4.1.1.154 server.log (jul ok, slf4j ok)
```
[2016-02-07T20:38:38.094+0900] [Payara 4.1] [INFO] [] [sample.logging.StartupLogger] [tid: _ThreadID=54 _ThreadName=admin-listener(4)] [timeMillis: 1454845118094] [levelValue: 800] [[
  julLogger init]]

[2016-02-07T20:38:38.095+0900] [Payara 4.1] [INFO] [] [sample.logging.StartupLogger] [tid: _ThreadID=54 _ThreadName=admin-listener(4)] [timeMillis: 1454845118095] [levelValue: 800] [[
  slf4jLogger init]]
```

Payara 4.1.1.161 server.log (jul ok, slf4j ng)
```
[2016-02-07T20:40:17.841+0900] [Payara 4.1] [INFO] [] [sample.logging.StartupLogger] [tid: _ThreadID=54 _ThreadName=admin-listener(4)] [timeMillis: 1454845217841] [levelValue: 800] [[
  julLogger init]]

[2016-02-07T20:40:17.842+0900] [Payara 4.1] [INFO] [] [] [tid: _ThreadID=54 _ThreadName=admin-listener(4)] [timeMillis: 1454845217842] [levelValue: 800] [[
  []]

[2016-02-07T20:40:17.843+0900] [Payara 4.1] [INFO] [] [] [tid: _ThreadID=54 _ThreadName=admin-listener(4)] [timeMillis: 1454845217843] [levelValue: 800] [[
  sample.logging.StartupLogger]]

[2016-02-07T20:40:17.843+0900] [Payara 4.1] [INFO] [] [] [tid: _ThreadID=54 _ThreadName=admin-listener(4)] [timeMillis: 1454845217843] [levelValue: 800] [[
  ] :]]

[2016-02-07T20:40:17.843+0900] [Payara 4.1] [INFO] [] [] [tid: _ThreadID=54 _ThreadName=admin-listener(4)] [timeMillis: 1454845217843] [levelValue: 800] [[
  slf4jLogger init]]
```

test code pom.xml
```
        <dependency>
            <groupId>org.slf4j</groupId>
            <artifactId>slf4j-api</artifactId>
            <version>1.7.14</version>
        </dependency>
        <dependency>
            <groupId>org.slf4j</groupId>
            <artifactId>slf4j-jdk14</artifactId>
            <version>1.7.14</version>
        </dependency>
```

test code StartupLogger.java
```
import javax.annotation.PostConstruct;
import javax.ejb.Singleton;
import javax.ejb.Startup;

@Singleton
@Startup
public class StartupLogger {
	private static java.util.logging.Logger julLogger = java.util.logging.Logger.getLogger(StartupLogger.class.getName());
	private static org.slf4j.Logger slf4jLogger = org.slf4j.LoggerFactory.getLogger(StartupLogger.class);
	
	@PostConstruct
	public void init() {
		julLogger.info("julLogger init");
		slf4jLogger.info("slf4jLogger init");
	}
}
```

**Comment by phillipross** [comment]
I suspect this might be related to an issue that I'm trying to figure out as well.
I believe what is happening in my case is that pax-logging was added between 4.1.1.154 and 4.1.1.161 (for a new version of metro?) and there is an slf4j binding that pax-logging is picking up which is overriding logging frameworks in deployed applications :disappointed: 

In a web app, it seems that setting the classloader delegate attribute to false in the glassfish-web.xml fixes the problem... but for EAR and ejb modules outside of webapps... I'm still trying to find a workaround.

**Comment by mulderbaba** [comment]
reproducer: [slf4j-test.zip](https://github.com/payara/Payara/files/120978/slf4j-test.zip)

**Comment by phillipross** [comment]
@mulderbaba I'm betting your test case will succeed if you edit the glassfish-web.xml file to set the delegate attribute value to "false"

**Comment by mulderbaba** [comment]
yes it will indeed @phillipross, but since this is a reproducer delegate is set to true. We are working on the alternatives to provide a solution.

**Comment by phillipross** [comment]
@mulderbaba yep, I understand.

I actually [rebuilt payara master branch w/o pax-logging](https://github.com/payara/Payara/issues/652#issuecomment-181151597) and tried your test case and it worked fine:
```
[2016-02-07T20:14:45.752-0500] [Payara 4.1] [INFO] [AS-EJB-00054] [javax.enterprise.ejb.container] [tid: _ThreadID=47 _ThreadName=admin-listener(5)] [timeMillis: 1454894085752] [levelValue: 800] [[
  Portable JNDI names for EJB StartupLogger: [java:global/slf4j-test/StartupLogger, java:global/slf4j-test/StartupLogger!fish.payara.test.StartupLogger]]]

[2016-02-07T20:14:45.803-0500] [Payara 4.1] [INFO] [] [org.jboss.weld.Event] [tid: _ThreadID=47 _ThreadName=admin-listener(5)] [timeMillis: 1454894085803] [levelValue: 800] [[
  WELD-000411: Observer method [BackedAnnotatedMethod] private org.glassfish.jersey.ext.cdi1x.internal.CdiComponentProvider.processAnnotatedType(@Observes ProcessAnnotatedType<Object>) receives events for all annotated types. Consider restricting events using @WithAnnotations or a generic type with bounds.]]

[2016-02-07T20:14:46.038-0500] [Payara 4.1] [INFO] [] [fish.payara.test.StartupLogger] [tid: _ThreadID=47 _ThreadName=admin-listener(5)] [timeMillis: 1454894086038] [levelValue: 800] [[
  julLogger init]]

[2016-02-07T20:14:46.039-0500] [Payara 4.1] [INFO] [] [fish.payara.test.StartupLogger] [tid: _ThreadID=47 _ThreadName=admin-listener(5)] [timeMillis: 1454894086039] [levelValue: 800] [[
  slf4jLogger init]]

[2016-02-07T20:14:46.063-0500] [Payara 4.1] [INFO] [AS-WEB-GLUE-00172] [javax.enterprise.web] [tid: _ThreadID=47 _ThreadName=admin-listener(5)] [timeMillis: 1454894086063] [levelValue: 800] [[
  Loading application [slf4j-test] at [/slf4j-test]]]

[2016-02-07T20:14:46.081-0500] [Payara 4.1] [INFO] [] [javax.enterprise.system.core] [tid: _ThreadID=47 _ThreadName=admin-listener(5)] [timeMillis: 1454894086081] [levelValue: 800] [[
  slf4j-test was successfully deployed in 463 milliseconds.]]

```

I'm convinced that adding pax-logging to satisfy JCL runtime dependency in the webservices modules was not the best solution.

**Comment by smillidge** [comment]
Pax logging was introduced to fix #538 but it looks like we've also brought in SL4J as an unforseen side effect. As @mulderbaba says looks like we may have to revisit this fix.

**Comment by smillidge** [comment]
Please try the latest Jenkins build http://www.payara.fish/upstream_builds.

**Comment by niblings** [comment]
That worked with the latest build. Thank you.

**Comment by smillidge** [comment]
Resolved in 4.1.1.161.1
